### PR TITLE
APPTS 9.2.0 release note

### PR DESCRIPTION
**Pull request by pkclark**
Thank you for your contribution to the Platform-Management-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new folder of topics>

### CODE REVIEW

File: content/en/docs/release_notes/09_02_00_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## New feature
+
+* Added a _Devices_ view in the _Account_ section for users using **Amplify Platform** for authentication allowing authorized devices to be viewed and managed.
+
+## Fixed issues
+
+* Fixed an issue where usage of **Cloud Capacity** subscription entitlements may fail to load on the _Manage Subscriptions_ service.
+* Fixed an issue where a banner on the _Users_ view indicating an org had unavailable roles assigned to its members may persist when switching between orgs.
+* Corrected an oudated _Connecting Environements_ documentation link on the _Platform Home_ Getting Started section.
```
Comment (sallylanda6) [inline content/en/docs/release_notes/09_02_00_relnotes.md:22]:
```suggestion
* Corrected an outdated _Connecting Environments_ documentation link on the _Platform Home_ Getting Started section.
```

File: content/en/docs/release_notes/09_02_00_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Hide_readingtime: true
+---
+
+## Platform Management 9.2.0 - 10 February 2022
+
+Platform Management 9.2.0 is a patch release which includes a new feature and several fixed issues.
+
+## New feature
+
+* Added a _Devices_ view in the _Account_ section for users using **Amplify Platform** for authentication allowing authorized devices to be viewed and managed.
```
Comment (sallylanda6) [inline content/en/docs/release_notes/09_02_00_relnotes.md:16]:
```suggestion
* Added a _Devices_ view in the _Account_ section for users using **{{% variables/platform_prod_name %}}** for authentication allowing authorized devices to be viewed and managed.
```

Comment (sallylanda6) [reply content/en/docs/release_notes/09_02_00_relnotes.md:16]: product variable changes it to Amplify platform

**Review by sallylanda6 — CHANGES_REQUESTED** [review]
See changes.

File: content/en/docs/release_notes/09_02_00_relnotes.md (hunk 2 above)
Comment (pkclark) [reply content/en/docs/release_notes/09_02_00_relnotes.md:16]: TIL those existed.

**Comment by pkclark** [comment]
Suggestions merged.

**Pull request merged**